src/Castle.Services.Logging.NLogIntegration/NLogLogger.cs

```diff
@@ -113,6 +122,84 @@ public virtual Core.Logging.ILogger CreateChildLogger(String name)
 			return Factory.Create(Logger.Name + "." + name);
 		}
 
+		/// <summary>
+		///   Logs a trace message.
+		/// </summary>
+		/// <param name="message"> The message to log </param>
+		public void Trace(string message)
+		{
+			Log(LogLevel.Trace, message);
+		}
+
+		/// <summary>
+		///   Logs a trace message.
+		/// </summary>
+		/// <param name="messageFactory"> Factory constructing lazily the message to log if the level is enabled </param>
+		public void Trace(Func<string> messageFactory)
+		{
+			if (IsTraceEnabled)
+			{
+				Log(LogLevel.Debug, messageFactory());
```

Comment:
`LogLevel.Debug` should be `LogLevel.Trace`, I think.

Reply:
Not sure how that happend - fixing.